log_outgoing_requests/validators.py

```diff
@@ -0,0 +1,54 @@
+from typing import TYPE_CHECKING, Any, Iterable, Optional
+
+if TYPE_CHECKING:  # pragma: nocover
+    from .models import OutgoingRequestsLogConfig
+
+
+def _validate_content_length(
+    content_length: Any, config: "OutgoingRequestsLogConfig"
+) -> bool:
```

Comment:
how is `content_length` known to be `str` in the calling function, but here it is `Any`?